File: webapp/src/test/java/uk/co/grahamcox/ccrpg/webapp/TestContext.java
package uk.co.grahamcox.ccrpg.webapp;

import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.ImportResource;

/**
 * Test context for Spring Integration tests
 */
@Configuration
@ImportResource({
    "classpath:/test-context.xml"
})
public class TestContext {
}

File: webapp/src/test/kotlin/uk/co/grahamcox/ccrpg/webapp/TestContext.kt
package uk.co.grahamcox.ccrpg.webapp

import org.springframework.context.annotation.ImportResource
import org.springframework.context.annotation.Configuration

/**
 * Test context for Spring Integration tests
 */
[Configuration]
[ImportResource(array("classpath:/test-context.xml"))]
open class TestContext()
